packages/node/src/indexer/types.ts

```diff
@@ -12,3 +12,12 @@ export interface BlockContent {
   extrinsics: SubstrateExtrinsic[];
   events: SubstrateEvent[];
 }
+
+export enum BlockEvents {
+  BlockProcessing = 'processing_block_event',
+}
+
+export interface BlockPayload {
+  data: { height: number; timestamp: number };
+  name: BlockEvents;
```

Comment:
doesn't make sense here, event name is `block.processing`, why you need another name filed in payload, when the value of it is only one choice.